apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentTabs/DocumentTriggers/Modal/TriggerTypeSelector.tsx

```diff
@@ -91,7 +91,10 @@ function IntegrationTriggerGroup({
   ) => void
   disabled?: boolean
 }) {
-  const { data, isLoading } = usePipedreamApp(integration.configuration.appName)
+  const { data, isLoading } = usePipedreamApp(
+    integration.configuration.appName,
+    { withConfig: true },
```

Comment:
I think this fat trigger is needed for the configuration panel on the triggers modal

Reply:
![](https://c.tenor.com/NMYouqcYwM8AAAAC/tenor.gif)